fhirAssembly/src/main/resources/assembly/com.vantiq.fhir.fhirConnection/procedures/com/vantiq/fhir/fhirService_searchCompartment.vail

```diff
@@ -0,0 +1,54 @@
+package com.vantiq.fhir
+
+import service com.vantiq.fhir.fhirService
+
+/**
+* Perform a search based on the compartment, id, & query provided
+*
+* Using the current source, run the search query for a particular compartment. If no method is provided,
+* we use GET. If search via POST is desired, provide "POST" as the method parameter.
+*
+* The compartment will be searched.  If the type parameter is provided, the results are restricted to entries of
+* that type.
+*
+* @param type String The FHIR Resource type to search
```

Comment:
nit: I think the first one should be "compartment" (and the description probably doesn't apply either).